vm_backtrace.c

```diff
@@ -65,13 +65,6 @@ calc_pos(const rb_iseq_t *iseq, const VALUE *pc, int *lineno, int *node_id)
             /* use pos-1 because PC points next instruction at the beginning of instruction */
             pos--;
         }
-#if VMDEBUG && defined(HAVE_BUILTIN___BUILTIN_TRAP)
```

Comment:
We end up in this branch fairly frequently with `pos == 0`. I've removed it for now, otherwise this will terminate any program right away.